packages/spotlight/src/spotlight.js

```diff
@@ -456,6 +456,26 @@ const Spotlight = (function () {
 		}
 	}
 
+	let throttleTimer = null;
+	let throttleClientX, throttleClientY;
+
+	function throttleMouseMove (e) {
```

Comment:
Can this throttle be implemented using `enact/core/Job` instead of a direct `setTimeout`?  Then we could compare the CPU% numbers from `start` (uses `setTimeout`) and `startRaf` (uses 'requestAnimationFrame`) and see if there is a noticeable difference.